### PR TITLE
DATA SCIENCE TEAM - Assessing Parking Availability Around  Café, restaurant, bistro seats,  Bar, tavern, pub - 30% Completed

### PR DESCRIPTION
**Pull request by ChathuzCodeCrafter**
Week 5 Pull Request for review Sprint 1
Description:  
This use case aims to evaluate the sufficiency of on-street parking around Café, restaurant, bistro seats, Bar, tavern, pub in Melbourne. 

Use Case Includes
-Use Case Introduction
-Import Data Using API  v2.1
-Data Wrangling
-Filtering datafreams
- Data Visualization using Bar Charts, Stack Bar Charts, Pie Charts
- Plot geolocation maps using folium

### CODE REVIEW

**Review by HarshDW09 — APPROVED** [review]
Looks good at the moment. For next sprint ,please ensure that the content is structured as a tutorial for the specific use case.Under each heading, provide a detailed description of the analysis being conducted. Utilize Markdown chunks to enhance readability and clarity.

Also , wait for the senior leader for full approval

**Review by kitkatch — APPROVED** [review]
Hi Chathu, 

Great work!! Agree with Harsh on the points that he's made.  Your use case is well structured, not a must for this sprint, but when completing your use case, please ensure that more instructions are used as this is supposed to be a tutorial for your reader.

**Pull request merged**
